conjur/config.py

```diff
@@ -29,25 +31,28 @@ class Config():
     FIELDS = [
         ('conjur_account', 'account', True),
         ('conjur_url', 'url', True),
-        ('cert_file', 'ca_bundle', False),
+        ('cert_file', 'ca_bundle', True),
     ]
 
     _config = {}
 
     def __init__(self, config_file=DEFAULT_CONFIG_FILE):
         # pylint: disable=logging-fstring-interpolation
-        logging.debug(f"Fetching connection details from filesystem {config_file}")
 
+        logging.debug(f"Fetching connection details from filesystem {config_file}")
         config = None
         with open(config_file, 'r') as config_fp:
             config = load(config_fp, Loader=Loader)
 
-        for config_field_name, attribute_name, mandatory in self.FIELDS:
-            if mandatory:
-                assert config_field_name in config
+        if not config:
+            for config_field_name, attribute_name, mandatory in self.FIELDS:
```

Comment:
could you please add comment here what and how are we checking? I didn't understood the FIELDS structure at first and what we are checking

Reply:
Yes this is very confusing logic. I will add a comment above. let me know if its more clear